deploy.py

```diff
@@ -160,6 +160,9 @@
 # `src/entry_point.rs`
 APP_HEAP_SIZE = 90000
 
+CARGO_TARGET_DIR = os.environ["CARGO_TARGET_DIR"] \
```

Comment:
`CARGO_TARGET_DIR = os.environ.get("CARGO_TARGET_DIR", "target")`

should also work? The second parameter is the default, if not present.

Reply:
Ah, yes. It looked kinda weird for being to long in the first place. I did not look at the python docs :smile:

Reply:
Ah fail, I thought you changed it already ^^